products/PDFmorph.rst

```diff
@@ -0,0 +1,117 @@
+.. include:: ../abbreviations.txt
+
+########
+PDFmorph
+########
+
+Overview
+========
+
+.. note::
+	The PDFmorph application has a Command Line Interface. If you are 
+	unfamiliar with the CLI, it is recommended that you consult online
+	resources and become somewhat familiar before using PDFmorph.
+
+PDFmorph is a Python software package designed to increase the insight researchers
+can obtain from measured atomic pair distribution functions (PDFs) in a model 
+independent way. The program was designed to help a researcher answer the question: 
+"has my material undergone a phase transition between these two measurements?"
+
+PDFmorph makes use of several data manipulation techniques to correct for benign
```

Comment:
@sbillinge I shortened the description a little bit here - please let me know if it's too detrimental to the content itself!

Reply:
It is good.  Make sure there is a pointer to the user manual documentation

btw, please make an issue to check all the docstrings.

Reply:
There's no manual in the diffpy.pdfmorph master, so I assume I'll have to link that once I make it, right? and will do on the docstrings issue! Would that be for this particular PR?

Reply:
package specific doc should go to diffpy.pdfmorph repo in a different PR.

You will need to create a special branch named `gh-pages so github will build the website for you automatically. We can talk about more details in person.

Reply:
Sounds good! I'll grab you when you come back

Reply:
> package specific doc should go to diffpy.pdfmorph repo in a different PR.
> You will need to create a special branch named `gh-pages so github will build the website for you automatically. 

actually the best would be to publish package docs on readthedocs.io, for example the docs for diffpy.structure are at https://diffpystructure.readthedocs.io/en/stable.

I have a TODO issue #22 to stop hosting manual at the gh-pages branch and move it over to RTD.  This should eventually happen for all diffpy packages.

Reply:
I'll get started on that! Thank you @pavoljuhas. So publishing on rtd is what we would want to do for the documentation for all of diffpy? Sounds like a decent project!

Reply:
> So publishing on rtd is what we would want to do for the documentation for all of diffpy?

Yes, I think it would save a lot of work and keep the manuals in sync with the source code.

Reply:
@pavoljuhas would you mind providing the source for the diffpy.structure that was published on readthedocs? That would be really helpful, thanks!

Reply:
The RTD page is rebuild on pushes to https://github.com/diffpy/diffpy.structure repo with the manual sources in `doc/manual`.  There is a small control file [.readthedocs.yml](https://github.com/diffpy/diffpy.structure/blob/df264a84139bc834424cc323d1bcfc1f9a4ca6cc/.readthedocs.yml) to setup the manual compilation at RTD.

Setting up an RTD build for PDFmorph should be an issue in the pdfmorph project.
Here we should create documentation links pointing to the RTD pages - once they are up.